docs/reference/index.rst

```diff
@@ -3,6 +3,13 @@ Reference
 =========
 
 .. toctree::
-   :maxdepth: 1
+    :maxdepth: 1
+
+    rubicon-objc
+    rubicon-objc-api
+    rubicon-objc-runtime
+    rubicon-objc-types
 
 This is the technical reference for public APIs provided by Rubicon.
+
+Note that the :mod:`rubicon.objc` package also contains other submodules not documented here. These are for internal use only and not part of the public API; they may change at any time without notice.
```

Comment:
They should; I think I just skipped writing the docs for `rubicon.objc.eventloop` because I really don't know much about `asyncio`. I've added a placeholder documentation page for the `.eventloop` module for now - it only lists the public classes/methods and pulls in their docstrings via autodoc where possible.